client/src/components/Toasts/Toast.test.js

```diff
@@ -79,7 +79,8 @@ describe('Notification Toast', () => {
     );
   });
 
-  it('shows toast including system information', () => {
+  // TODO v3: update test after file operations are implemented and we have updated test fixtures
```

Comment:
Just to keep the todo string consistent, for searchability later
```suggestion
  // TODOv3: update test after file operations are implemented and we have updated test fixtures
```